src/GraphQL.Conventions/Types/Descriptors/TypeNames.cs

```diff
@@ -12,7 +12,7 @@ public static class TypeNames
 
         public const string Boolean = "Boolean";
 
-        public const string Id = "ID";
+        public const string Identity = "Identity";
```

Comment:
Why this change?

https://github.com/graphql-dotnet/graphql-dotnet/blob/master/src/GraphQL/Types/Scalars/IdGraphType.cs

Reply:
There was a clash between both IdGraphType (the conventions one and the graphQL.Net one). However I did that more than a year ago and I can't remember why I did that. Maybe the Conventions.IdGraphType is redundant now?
I will revert the changes around this so we can move forward with this PR and we can do more test around the IdGraphType.

Reply:
The error is:
Unable to register GraphType 'GraphQL.Conventions.Adapters.Types.IdGraphType' with the name 'ID' the name 'ID' is already registered to 'GraphQL.Types.IdGraphType'
Any idea what would be the best way to handle this ?

Reply:
https://github.com/graphql-dotnet/graphql-dotnet/issues/1952

Reply:
I looked at this issue differently, Instead of having an overload of IdGraphType, I added a TypeConverter on the Id class (and support in the primitive wrapper).

Reply:
Thanks :+1: